### PR TITLE
London Class 7 - Deniz Kelemence - Git Session- Week 1

### PR DESCRIPTION
**Pull request by ASLANDENIZ**
# Your Details

- Your Name: Deniz Kelemence
- Your City: London
- Your Slack Name: Deniz Kelemence

# Homework Details

- Module: Git Session
- Week:1

### CODE REVIEW

**Review by aishak05 — COMMENTED** [review]
Great work Deniz! I especially like how you have elaborated further on answers such as in questions 10 and 15 to explain how the concept works.

**Comment by jmodiale** [comment]
Nice one Deniz, I can say you've passed this with flying colors!

File: homework.md [hunk 1]
```diff
@@ -4,77 +4,76 @@
...
 
-When people want to show off code to each other they can put it on Github
+Version control, collaboration, showcase your work
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Creating a “fork” is producing a personal copy of someone else's project
```
Comment (ademalagoz) [inline homework.md:58]: Maybe we can add a little words to last sentence.
Creating a “fork” is producing a personal copy of someone else's project **to my Github.**

File: homework.md [hunk 2]
```diff
@@ -4,77 +4,76 @@
...
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```
Comment (Jpsi) [inline homework.md:17]: What's the purpose of this change?

**Review by Jpsi — COMMENTED** [review]
Well done!

Tip: you can copy definitions of JavaScript, Pull Request, commit, etc, from the Internet, but that slows down your own learning. Trying to use your own words to convey a meaning (after you've looked up the definition, of course) takes slightly more time in the short term, but saves you more time in the long term.
